WordPress/src/main/java/org/wordpress/android/ui/CommentFullScreenDialogFragment.kt

```diff
@@ -55,11 +59,30 @@ class CommentFullScreenDialogFragment : Fragment(), CollapseFullScreenDialogCont
             reply.setText(it.getString(EXTRA_REPLY))
             reply.setSelection(it.getInt(EXTRA_SELECTION_START), it.getInt(EXTRA_SELECTION_END))
             viewModel.init()
+
+            setupSuggestionServiceAndAdapter(it.getSerializable(EXTRA_SITE_MODEL) as SiteModel, reply)
         }
 
         return layout
     }
 
+    private fun setupSuggestionServiceAndAdapter(mSite: SiteModel, reply: SuggestionAutoCompleteText) {
```

Comment:
Is there a reason why you passed `reply` here and not access `this.reply` directly?

Reply:
I tried a bunch of things to refer to the view that didn't work, and things got weird and decided to just pass it as a parameter. I changed this up and can confirm that using `this.reply` works, though, thanks!.